noir-projects/aztec-nr/value-note/src/value_note.nr

```diff
@@ -1,30 +1,24 @@
 use dep::aztec::{
-    generators::{Ga1 as G_amt, Ga2 as G_npk, Ga3 as G_rnd, G_slot},
-    protocol_types::{
-    traits::Serialize, constants::GENERATOR_INDEX__NOTE_NULLIFIER, hash::poseidon2_hash_with_separator,
-    point::{Point, POINT_LENGTH}
-},
-    note::{note_header::NoteHeader, note_interface::NoteInterface, utils::compute_note_hash_for_nullify},
+    protocol_types::{traits::Serialize, constants::GENERATOR_INDEX__NOTE_NULLIFIER, hash::poseidon2_hash_with_separator},
+    macros::notes::note,
+    note::{note_header::NoteHeader, note_interface::NullifiableNote, utils::compute_note_hash_for_nullify},
     oracle::unsafe_rand::unsafe_rand, keys::getters::get_nsk_app, context::PrivateContext
 };
-use dep::std::{embedded_curve_ops::multi_scalar_mul};
-use std::hash::from_field_unsafe;
 
 global VALUE_NOTE_LEN: u32 = 3; // 3 plus a header.
```

Comment:
I see that this value is now unused when generating the note interface as the macro is smart enough to figure the length out. Is it the expectation that we nuke this?

Reply:
Hopefully yes! It is used in some places though, but private to the `value_note` crate. Hopefully we will have viz on globals soon.